modules/ROOT/partials/configuration/sidebar_show.adoc

```diff
@@ -0,0 +1,27 @@
+[[sidebar_show]]
+== `+sidebar_show+`
+
+NOTE: This option is only available for TinyMCE 6.1 and later.
```

Comment:
As noted in all other PRs I've reviewed so far, this shouldn't be done this way. See https://github.com/tinymce/tinymce-docs/pull/2471#discussion_r918548089. We'll need to get someone to make that change and merge first so we can all reuse it.